src/main/resources/errors.yaml

```diff
@@ -392,7 +398,45 @@ request-errors:
       
       The API appended the CQL optional `ALLOW FILTERING` to the query and retried.
       
-      ${SNIPPET.INEFFICIENT_QUERY}
+      ${SNIPPET.INEFFICIENT_FILTER}
+
+
+  - scope: WARNING
+    code: IN_MEMORY_SORTING_DUE_TO_NON_PARTITION_SORTING
+    title: Sorting by non partition sorting columns
+    body: |-
+      The command used columns in the sort clause that are not part of the partition sorting, and so the query was sorted in memory.
+            
+      The table ${keyspace}.${table} has the partition sorting columns: ${partitionSorting}.
+      The command sorted on the columns: ${sortColumns}.
```

Comment:
See another comment in TableSortClauseResolver.java, ${sortColumns} has been filtered in stream, so not actually columns we use for in-memory sort.